api/src/data/models/action-log-model.ts

```diff
@@ -0,0 +1,24 @@
+export interface ActionLog {
+    id: number;
+    action_id: number; // nullable or not isnt specified
```

Comment:
With MSSQL if you don't specify `notNullable` the column is nullable.
See https://knexjs.org/guide/schema-builder.html#nullable
> Default on column creation ...